orchestrate/spark_example/orchestrate.yml

```diff
@@ -0,0 +1,29 @@
+language: python3.6
+name: Orchestrate Spark Example (python)
+install:
+  -  pip install -r requirements.txt
+  -  wget --quiet http://apache.claz.org/spark/spark-2.4.0/spark-2.4.0-bin-hadoop2.7.tgz
+  -  tar -xvf spark-2.4.0-bin-hadoop2.7.tgz
+  -  rm spark-2.4.0-bin-hadoop2.7.tgz
+  -  apt-get update
+  -  apt-get install -y default-jdk
+  -  export SPARK_MASTER_HOST=localhost
+  -  export SPARK_MASTER_PORT=7077
+  -  spark-2.4.0-bin-hadoop2.7/sbin/start-master.sh
+  -  spark-2.4.0-bin-hadoop2.7/sbin/start-slave.sh spark://localhost:7077
+run:
+  -  spark-2.4.0-bin-hadoop2.7/bin/spark-submit model.py
+optimization:
+  metrics:
+    - name: accuracy
+  parameters:  # Fill in parameters to optimize
+    - name: l1_ratio
```

Comment:
we could use this as a chance to just run a basic franke function optimization, with parameters x and y. See how it's done here: https://app.sigopt.com/docs/overview/python (franke function evaluation code is already implemented)

Reply:
Although on second thought, this may not work so well with the parallelization